docs/additional_examples.md

```diff
@@ -0,0 +1,121 @@
+## MDBCI usage
```

Comment:
The name of the file is wrong as it does not provide additional examples, but provide a complex template and describes the MDBCI operation in some details.